superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -253,6 +257,12 @@ export default function DataSourcePanel({
     setInputValue('');
   }, [columns, datasource, metrics]);
 
+  const sortCertifiedFirst = (slice: ColumnMeta[]) => {
```

Comment:
I think here for brevity you could just sort the slice array, something like:

`slice.sort((a, b) => (b. is_certified - a. is_certified));`

Reply:
Agreed

Reply:
Does this retain the order within the certified vs non-certified groups? Doing it like it's done here should at least ensure the order is retained.

Reply:
@villebro I think the order is retained. Here is a short demo https://jsfiddle.net/9c0fjpwv/

Reply:
Yes this works thanks for the suggesiton.